style.css

```diff
@@ -6,6 +6,10 @@
   padding: 0;
 }
 
+body {
+  overflow-x: hidden;
+}
+
```

Comment:
- [OPTIONAL] Your menu navigation works great, :+1: :+1: I'll suggest you add a smooth scroll to your portfolio, it will make scrolling to different sections of the page when a user clicks on a menu item smooth. You can achieve this by adding this line of CSS at the top of your CSS file. `html {
  scroll-behavior: smooth;
}`. :+1: :smile: